root_draft_metadata_ecr.tf

```diff
@@ -0,0 +1,25 @@
+module "draft_metadata_validator_lambda_ecr" {
+  source          = "./da-terraform-modules/lambda"
+  function_name   = "tdr-draft-metadata-validator-2-${local.environment}"
```

Comment:
Could you rename the function: `...-validator-v2-...` Its the pattern used for other revised Lambdas on TDR eg `tdr-yara-av-v2-intg`

Reply:
changed